### ISSUE TITLE
CustomDialog MaxWidth ?

### ISSUE THREAD

**Issue by Kuirak** [timheuer/callisto#122]
I am currently working with your custom dialog control and want to add a gridview to it.  I discovered that you set  a MaxWidth="680"  which causes ugly clipping of the  gridview content  and  the scrollbar. 

Why did you introduce this value?  Doesn't  the automatic width handle that  everything is working  with different resolutions?

Is there a way to overwrite  the customdialog style to get rid of the maxwidth or is it really needed?

Greets Kuirak

**Comment by timheuer** [comment]
This width is per the UI guidelines for windows. Perhaps the correct fix here is actually padding to anticipate the scroll area. 

-th

**Comment by Kuirak** [comment]
Hi tim,
I did some testing and can't get the padding  to work with the Current set up of UIelements. I also tried different values for maxwidth and realized  around  900 is a good value if  you want to have  some bigger content  like  a gridview with tiles  in  the dialog. It still working  on  surface sized screen, even in 2/3 mode. During testing I discovered that snapped view  isnt working correct even the backbutton isn't displayed any more (the title string has around 25 chars).

We need to find a better  way to handle different width or a way to manipulate the source style theme.

regards
